model/FeatureFlag/UniqueNumericQtiIdentifierClientConfig.php

```diff
@@ -27,14 +27,15 @@
 
 class UniqueNumericQtiIdentifierClientConfig implements FeatureFlagConfigHandlerInterface
 {
+    public const QTI_ID_PATTERN = '/^\\d{9}$/';
```

Comment:
Is that correct pattern with `\\d` ? Should it be `\d` ?

Reply:
Should we update it with Normalize string?

Reply:
It should work. And we should keep it as regexp that will accept 9 digits. It is `UniqueNumericQtiIdentifier` validation.